### PR TITLE
Change text

### PR DESCRIPTION
**Pull request by hectoruch**
- https://www.pivotaltracker.com/story/show/147843837

### CODE REVIEW

**Comment by hectoruch** [comment]
ranking

**Comment by hectoruch** [comment]
@pjosh 
@simaob

File: app/assets/javascripts/countries/views/widgets/TreeCoverLossRankingView.js [hunk 1]
```diff
@@ -62,35 +63,83 @@ define([
...
+            break;
+          }
+        }
+      } else {
+        for (var x = 0; x < this.data.length; x++) {
+          item = this.data[x];
+          if (item.adm1 === parseInt(this.region)) {
+            isoIndex = x;
+            break;
+          }
```
Comment (pjosh) [inline app/assets/javascripts/countries/views/widgets/TreeCoverLossRankingView.js]: We have duplicate code here. I think this code works well too:
```
for (var x = 0; x < this.data.length; x++) {
  item = this.data[x];
  
  if ((this.region === 0 && item.iso === this.iso) ||
    item.adm1 === parseInt(this.region)) {
    isoIndex = x;
    break;
  }
}
```

File: app/assets/javascripts/countries/views/widgets/TreeCoverLossRankingView.js [hunk 2]
```diff
@@ -62,35 +63,83 @@ define([
...
+          this.list.push(this.data[isoIndex]);
+
+          if (isoIndex < this.data.length) {
+            this.list.push(this.data[isoIndex + 1]);
+          }
+        } else {
+          this.list.push(this.data[isoIndex]);
+          this.list.push(this.data[isoIndex + 1]);
+          this.list.push(this.data[isoIndex + 2]);
+        }
```
Comment (pjosh) [inline app/assets/javascripts/countries/views/widgets/TreeCoverLossRankingView.js]: The `if` as the same as the `else`

File: app/assets/javascripts/countries/views/widgets/TreeCoverLossRankingView.js [hunk 3]
```diff
@@ -62,35 +63,83 @@ define([
...
+        if (this.region === 0) {
+          item.selected = item.iso === this.iso;
+        } else {
+          item.selected = item.adm1 === parseInt(this.region);
+        }
+        if (this.region != 0) {
+          this.titleText = 'Regions ranking';
+        } else {
+          this.titleText = 'Tree cover country ranking 2010';
+        }
```
Comment (pjosh) [inline app/assets/javascripts/countries/views/widgets/TreeCoverLossRankingView.js]: Here we have the same conditionals many times. It's better for reading and performance if we encapsulated them on a single `if/else`.

Comment (hectoruch) [inline app/assets/javascripts/countries/views/widgets/TreeCoverLossRankingView.js]: Yes!

**Comment by hectoruch** [comment]
This change is here.

image

**Pull request merged**